apis/apis.json

```diff
@@ -386,7 +386,8 @@
 
   {
     "id": "Google.Cloud.OsLogin.V1Beta",
-    "productName": "Google Cloud OS Login API",
+    "productName": "Google Cloud OS Login API"
```

Comment:
Missing a comma?

Reply:
That would explain the CI failure, yes :) Thanks.